scripts/build_libsnowflakeclient.bat

```diff
@@ -2,56 +2,98 @@
 :: Build Snowflake Client library
 ::
 @echo off
+set scriptdir=%~dp0
+for /f "tokens=3" %%A in ('findstr SF_API_VERSION %scriptdir%..\include\snowflake\version.h') do @set V=%%A
+set libsnowflakeclient_version=%v:"=%
+call %*
+goto :EOF
+
+:get_version
+	set version=%libsnowflakeclient_version%
+	goto :EOF
```

Comment:
One of the major changes in this PR is added labels: `get_version` and `build` so that the caller can get the version aside from running the build.